src/main/java/com/faforever/client/game/GameTileController.java

```diff
@@ -78,6 +83,10 @@ private void onGamePropertyChanged() {
     JavaFxUtil.runLater(() -> {
       gameTitleLabel.setText(StringUtils.normalizeSpace(game.getTitle()));
       hostLabel.setText(game.getHost());
+      playerService.getPlayerByNameIfOnline(game.getHost()).map(PlayerBean::getAvatar).ifPresent(avatar -> {
+        avatarImageView.setImage(avatarService.loadAvatar(avatar));
+        Tooltip.install(avatarImageView, new Tooltip(avatar.getDescription()));
+      });
```

Comment:
Do not load the avatar images in the application thread it should be moved outside the run later